### PR TITLE
Update ModuleNotFound.md

### PR DESCRIPTION
**Pull request by ktec**
This is a little better than when I found it...

### CODE REVIEW

File: errors/ModuleNotFound.md [hunk 1]
```diff
@@ -10,12 +10,24 @@ module ShortFailingExample where
 
 ## Cause
 
-Explain why a user might see this error.
+You might see this error because you have tried to import a Module that is not available on your system.
 
 ## Fix
 
 - Suggest possible solutions.
```
Comment (garyb) [inline errors/ModuleNotFound.md]: We could probably remove this line now too :wink:

File: errors/ModuleNotFound.md [hunk 2]
```diff
@@ -10,12 +10,24 @@ module ShortFailingExample where
 
 ## Cause
 
-Explain why a user might see this error.
+You might see this error because you have tried to import a Module that is not available on your system.
```
Comment (garyb) [inline errors/ModuleNotFound.md]: Probably worth mentioning that it could be a typo / incorrect name too!

Comment (hdgarrood) [inline errors/ModuleNotFound.md]: I think this sentence makes it sound like some modules aren't available on some operating systems; I think a better wording would be "the compiler was unable to find the module you tried to import" or something.

File: errors/ModuleNotFound.md [hunk 3]
```diff
@@ -10,12 +10,24 @@ module ShortFailingExample where
...
 
 ## Fix
 
 - Suggest possible solutions.
 
+Its likely you need to install the package which contains the module you are trying to use.
```
Comment (hdgarrood) [inline errors/ModuleNotFound.md]: This should be "it's"

File: errors/ModuleNotFound.md [hunk 4]
```diff
@@ -10,12 +10,24 @@ module ShortFailingExample where
...
+
+A great place to go looking is https://pursuit.purescript.org where you can search for the Module by name, and find out which package you need to install.
+
+To install you'll want to use something like this:
+
+```
+bower install purescript-strings --save
+```
+
+You may need to use a different command, depending on how you have installed Purescript.
```
Comment (hdgarrood) [inline errors/ModuleNotFound.md]: The specific command doesn't really depend on how you have installed PureScript, but rather how your project is set up, I think.

File: errors/ModuleNotFound.md [hunk 5]
```diff
@@ -10,12 +10,24 @@ module ShortFailingExample where
...
-Explain why a user might see this error.
+You might see this error because you have tried to import a Module that is not available on your system.
 
 ## Fix
 
 - Suggest possible solutions.
 
+Its likely you need to install the package which contains the module you are trying to use.
+
+A great place to go looking is https://pursuit.purescript.org where you can search for the Module by name, and find out which package you need to install.
```
Comment (hdgarrood) [inline errors/ModuleNotFound.md]: Can you use a lowercase M for module?

**Comment by hdgarrood** [comment]
I'm going to make those changes myself and merge this. Thank you!

**Comment by hdgarrood** [comment]
See #91.

**Pull request closed without merging**